src/data-root.ts

```diff
@@ -0,0 +1,46 @@
+import localforage from 'localforage'
+
+import * as core from './core'
+import * as dns from './dns'
+import { CID } from './ipfs'
+import { UCAN_STORAGE_KEY } from './common'
+
+
+/**
+ * Get the CID of a user's data root.
+ */
+export async function dataRoot(username: string): Promise<CID> {
+  try {
+    // TODO: This'll be `files.${username}.fission.name` later
+    return await dns.lookupDnsLink(`${username}.fission.name`)
+  } catch(err) {
+    throw new Error("Could not locate user root in dns")
+  }
+}
+
+/**
+ * Update a user's data root.
+ */
+export const updateDataRoot = async (props: {
+  apiEndpoint?: string
+  apiDid?: string
+  cid: CID | string
+}): Promise<any> => {
```

Comment:
Maybe give this a type? `any`s kinda blow apart TS type system. I'm trying to steer clear of them except for in runtime type checks (and even then I'd like to start using `unknown`, but I don't think I'm doing it correctly cause TS always complains :stuck_out_tongue:)

Reply:
actually, does this even need to return anything?

Reply:
No it doesn't. I wanted to return `Promise<void>` or something, but I don't know how to do that. Any ideas?

Reply:
I'm pretty sure you can do just that^^ 

if it throws a fit just an empty
```ts
return
```
at the end of the fn should do it

Reply:
Oh ok, neato, fixed it by using `await` and no return.